src/events/components/ImageView/index.tsx

```diff
@@ -46,14 +46,13 @@ const isPopulated = (imageEvents: IState) => {
 };
 
 export const ImageView = ({  }: IProps) => {
-  const { eventList, updateEventList } = useContext(EventsRepo);
+  const dispatch = useDispatch();
+  const eventList = useSelector((state) => eventSelectors.selectAll(state));
+  // const { eventList, updateEventList } = useContext(EventsRepo);
 
   /** Fetch events to repo on mount */
```

Comment:
This comment may not be strictly correct anymore?

Reply:
True, it's store and not repo now 😄